service/worker/archiver/pump.go

```diff
@@ -37,42 +37,45 @@ type (
 	// PumpResult is the result of pumping requests into request channel
 	PumpResult struct {
 		PumpedHashes          []uint64
-		UnhandledCarryover    []ArchiveRequest
+		UnhandledCarryover    []interface{}
```

Comment:
Is this backwards compatible?

Reply:
Tested locally. It's backward compatible.